docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -40,7 +37,7 @@ The Codewind installation pulls the following images that form the Codewind back
 
 The Codewind installation includes two parts:
 
-1. The Eclipse plug-in installs when you install Codewind from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind) or when you install by searching in the `Eclipse Extensions` view.
+1. The Eclipse plug-in installs when you install Codewind from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind). Or from the Eclipse IDE, you can go to `Help -> Eclipse Marketplace` then search for Codewind. 
```

Comment:
The format `**Help**>**Eclipse Marketplace**` would match the current style guidelines.